src/Keboola/DbExtractor/Logger.php

```diff
@@ -4,7 +4,7 @@
 
 namespace Keboola\DbExtractor;
 
-use Keboola\DbExtractor\Logger\LineFormatter;
+use Monolog\Formatter\LineFormatter;
```

Comment:
ten logger by se mel cely vyhodit je to jen zpetne kompatibilni reimplementace https://github.com/keboola/php-component/blob/master/src/Logger.php a pokud se budou delat nekompatibilni zmeny, tak tady neni na nic potreba